pycypher/src/Makefile.am

```diff
@@ -0,0 +1,49 @@
+generated_bindings = getters.py node_types.c operators.c props.c
+
+.NOTPARALLEL: $(generated_bindings)
+$(generated_bindings): $(top_srcdir)/lib/src/cypher-parser.h.in
+	$(PYTHON) $(top_srcdir)/build-aux/pycypher/generate-bindings.py < $<
+
+pkgpyexec_LTLIBRARIES = pycypher.la
+pycypher_la_SOURCES = \
```

Comment:
I don't know. Certainly AM_PATH_PYTHON (https://www.gnu.org/software/automake/manual/html_node/Python.html) suggests using these, but I'm unclear on how pip or setuptools work yet, so this was just an experiment.

Reply:
BTW - I did get it to build a python module using these targets, but I couldn't get the testing to work. It seemed to fail in loading the shared library, due to a runtime linking issue (it expected the libcypher-parser.so to be already installed, rather than using the version compiled from the source tree).

Reply:
You probably need to adjust LD_LIBRARY_PATH

Reply:
Unfortunately, that doesn't work. The bindings library has an RPATH compiled into it, based on the target install location of libcypher-parser.so. I need to figure out how to adjust that so that it can be run against the built version.

BTW - I found that the `check_pycypher.sh` was actually picking up the installed version of libcypher-parser, not the version from the source tree.

Reply:
So you compiled bindings using autotools and not `setup.py build_ext` ? I would suggest to just call the latter from Makefile.am, then the `.so` should not have any RPATH compiled inside it.

Reply:
It definitely should not have RPATH compiled in it.

Reply:
I compiled both ways on my macos development system and both included RPATH into the bindings.so. I need to do some research to see how to handle that when doing trying to run tests locally.